packages/caip/src/constants.ts

```diff
@@ -1,15 +1,15 @@
-import { AssetNamespace } from './assetId/assetId'
-import { ChainNamespace, ChainReference } from './chainId/chainId'
+import { AssetId, AssetNamespace } from './assetId/assetId'
+import { ChainId, ChainNamespace, ChainReference } from './chainId/chainId'
 
-export const btcAssetId = 'bip122:000000000019d6689c085ae165831e93/slip44:0'
-export const ethAssetId = 'eip155:1/slip44:60'
-export const cosmosAssetId = 'cosmos:cosmoshub-4/slip44:118'
-export const osmosisAssetId = 'cosmos:osmosis-1/slip44:118'
+export const btcAssetId: AssetId = 'bip122:000000000019d6689c085ae165831e93/slip44:0'
+export const ethAssetId: AssetId = 'eip155:1/slip44:60'
+export const cosmosAssetId: AssetId = 'cosmos:cosmoshub-4/slip44:118'
+export const osmosisAssetId: AssetId = 'cosmos:osmosis-1/slip44:118'
 
-export const ethChainId = 'eip155:1'
-export const btcChainId = 'bip122:000000000019d6689c085ae165831e93'
-export const cosmosChainId = 'cosmos:cosmoshub-4'
-export const osmosisChainId = 'cosmos:osmosis-1'
+export const ethChainId: ChainId = 'eip155:1'
+export const btcChainId: ChainId = 'bip122:000000000019d6689c085ae165831e93'
+export const cosmosChainId: ChainId = 'cosmos:cosmoshub-4'
+export const osmosisChainId: ChainId = 'cosmos:osmosis-1'
 
 export const CHAIN_NAMESPACE = {
```

Comment:
I understand this comment is outside of your changes but this caught my attention.

`CHAIN_NAMESPACE` is essentially an enum and I'd consider removing "_NAMESPACE" since `namespace` is a keyword and `Chain` would suffice.

Reply:
@GMSteuart I get where you're coming from here, though in this case the enum represents an array of possible values of type `ChainNamespace`.

The "namespace" nomenclature is one we've chosen to represent the parts of a `ChainId`, which is `ChainNamespace:ChainReference`.

Similarly, we have `AssetNamespace` and `AssetReference`, along with an `AssetId` that encompasses them both.

Agree that there might be an argument for not using a TypeScript keyword in the name, but that ship has sailed.